fluent-syntax/src/serializer.js

```diff
@@ -28,7 +28,7 @@ export default class FluentSerializer {
     }
 
     for (const entry of resource.body) {
-      if (entry.types !== 'Junk' || this.withJunk) {
+      if (entry.type !== 'Junk' || this.withJunk) {
```

Comment:
A nasty typo but fortunately didn't impact many tests. It made the serializer serialize Junks verbatim as they appear in the parsed text. Input magically always matched the output...